packages/fx-core/templates/plugins/resource/simpleauth/bicep/simpleAuth.template.bicep

```diff
@@ -0,0 +1,64 @@
+param sku string
+param simpleAuthServerFarmsName string
+param simpleAuthWebAppName string
+param tenantId string
+param AADClientId string
+@secure()
+param AADClientSecret string
+param applicationIdUri string
+{{#contains 'fx-resource-frontend-hosting' plugins}}
+
+param frontendHostingStorageEndpoint string
+{{/contains}}
+
+var oauthAuthorityHost = environment().authentication.loginEndpoint
+var aadMetadataAddress = uri(oauthAuthorityHost, '${tenantId}/v2.0/.well-known/openid-configuration')
+var oauthAuthority = uri(oauthAuthorityHost, tenantId)
+var teamsAadIds = '1fec8e78-bce4-4aaf-ab1b-5451cc387264;5e3ce6c0-2b1f-4285-8d4b-75ee78787346'
+
+resource simpleAuthServerFarms 'Microsoft.Web/serverfarms@2020-06-01' = {
+  name: simpleAuthServerFarmsName
+  location: resourceGroup().location
+  sku: {
+    name: sku
+  }
+  kind: 'app'
+  properties: {
+    reserved: false
```

Comment:
is this required? if not can we just leave the properties empty?

Reply:
Yes, it is required.